<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaginaPrincipal</class>
 <widget class="QMainWindow" name="PaginaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibracion App</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_Run">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Titulo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 38, 38)
</string>
    </property>
    <property name="text">
     <string>PROGRAMA DE CALIBRACIÓN</string>
    </property>
   </widget>
   <widget class="QFrame" name="F1_Usuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_UsuarioId">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuario Id:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="value_le_UsuarioId">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>142</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="value_le_CalibracionId">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>142</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_CalibracionId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>96</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibración Id:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_UsuarioIcon">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Media/user-icon2p.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="F2_Camara">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_CamaraIcon">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Media/camera-icon2p.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lb_CamaraId">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cámara Id:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="value_cb_CamaraId">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)
</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="F4_Comentarios">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>571</width>
      <height>221</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_Comentarios">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Comentarios: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="value_lb_Comentarios">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>451</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="F3_Chessboard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_NumeroCuadros">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Nº Cuadros: </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="value_sb_NumeroCuadrosHorizontales">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_NumeroCuadrosHorizontales">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>82</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>(Horizontales)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="value_sb_NumeroCuadrosVerticales">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_NumeroCuadrosVerticales">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>(Verticales)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_SizeCuadros">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>T Cuadros:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="value_sb_SizeAltoCuadros">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_SizeAltoCuadros">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>(Alto, mm)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="value_sb_SizeAnchoCuadros">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_SizeAnchoCuadros">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>82</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>(Ancho, mm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_NumeroImagenes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nº Imágenes: </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="value_sb_NumeroImagenes">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ChessboardPatternIcon">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Media/chessboard-icon2p.jpeg</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
